README.md

```diff
@@ -177,7 +177,29 @@ all the derivatives are processed uniformly.
 
 ## Run the diffusion MRI analysis pipeline
 
-@Arnaud
+Github used to combine flows: https://github.com/scilus/combine_flows
+
+### Tractoflow
+```
+nextflow tractoflow -r 2.4.0 --bids anat/ --fs /data/neuromod/DATA/cneuromod.processed/smriprep/sourcedata/freesurfer/ --max_dti_shell_value 1550 -profile use_cuda,ABS -with-docker scilus/scilus:1.4.0 -resume --run_gibbs_correction True
```

Comment:
This hardcoded path (`/data/neuromod/DATA/cneuromod.processed/smriprep/sourcedata/freesurfer/`) should be explained; what's in it and how do we get it?

Reply:
The instructions you wrote also appear to differ from the official TractoFlow doumentation; @arnaudbore can you please also describe the environment setup steps prior to the commands you listed, so that they can be run?